src/transformers/models/fuyu/modeling_fuyu.py

```diff
@@ -210,7 +210,11 @@ def gather_continuous_embeddings(
     @add_start_docstrings_to_model_forward(FUYU_INPUTS_DOCSTRING)
     def forward(
         self,
-        sequences,
+        input_ids: torch.LongTensor = None,
+        image_patches: torch.Tensor = None,  # [batch_size, num_total_patches, patch_size_ x patch_size x num_channels ]
```

Comment:
What if inputs have different numbers of images in a batch? Not sure if they can/should be concatenated in a tensor. Consider List_of_batch[List_of_images[tensor(num_total_patches, patch_size_ x patch_size x num_channels)]]?